### PR TITLE
implemented stream

### PR DESCRIPTION
**Pull request by Raynos**

### CODE REVIEW

File: stream.js [hunk 1]
```diff
@@ -1,66 +1,28 @@
...
 
-// Define implementation of `accumulate` for node streams
-// so that they can be reduced as any other data structures
-// representing collections.
-reduce.define(Stream, function(stream, next, initial) {
-  if (stream.readable === false) return next(end, initial)
-  var result = initial
-  var ended = false
+function stream(reducible) {
+  var foo = "bar"
```
Comment (Gozala) [inline stream.js:12]: ???

Comment (Raynos) [inline stream.js:12]: I missed that line.

I needed a dummy line to get my text editor to "detect" 2 line

File: README.md [hunk 2]
```diff
@@ -53,6 +53,24 @@ that ary lazy and do work only after calling `reduce` on them, node streams
```
Comment (Gozala) [inline README.md:52]: Could you please avoid change in file name.

**Comment by Gozala** [comment]
After merging #5 this no longer can be merged because of conflicts. Mind resolving those. Also I would prefer not changing `stream - index` and creating another `stream` since it messes up history. I think it would be better to just add stream function to the same module.

**Comment by Raynos** [comment]
Closing this due to staleness

**Pull request closed without merging**